docs/content/0.1.0/docs/deploying-dj/notifications.md

```diff
@@ -81,4 +80,33 @@ notify(event)
 ```
 
 By customizing the `get_notifier` dependency, you can tailor the notification system to suit your specific needs, such
-as integrating with third-party services or implementing advanced notification configuration or logic.
+as integrating with third-party services or implementing advanced notification configuration or logic. Below are the different
+types of entities and activities that DataJunction will pass through the notification dependency.
+
+### Entity types
+
+- ATTRIBUTE
+- AVAILABILITY
+- BACKFILL
+- CATALOG
+- COLUMN_ATTRIBUTE
+- DEPENDENCY
+- ENGINE
+- LINK
+- MATERIALIZATION
+- NAMESPACE
+- NODE
+- PARTITION
+- QUERY
+- TAG
+
+### Activity types
```

Comment:
The docs still look up to date but I decided to list out the entity types and activity types at the end for convenience. I find myself looking these enums up in the code enough times that it would be useful to have on the docs site. :P